qutebrowser/config/configdata.py

```diff
@@ -1683,7 +1683,7 @@ def data(readonly=False):
     (re.compile(r'^scroll ([-\d]+ [-\d]+)$'), r'scroll-px \1'),
 
     (re.compile(r'^search *;; *clear-keychain$'), r'clear-keychain ;; search'),
-    (re.compile(r'^leave-mode$'), r'clear-keychain ;; leave-mode'),
+    (re.compile(r'^clear-keychain *;; *leave-mode$'), r'leave-mode'),
```

Comment:
You need to adjust `TestKeyConfigParser.test_migrations` in `tests/unit/config/test_config.py` accordingly.